source/Glimpse.Ado/AlternateType/GlimpseDbProviderFactory.cs

```diff
@@ -31,9 +33,27 @@ public override bool CanCreateDataSourceEnumerator
 
         private TProviderFactory InnerFactory { get; set; }
 
+        private static bool ShouldNotWrap()
+        {
+            if (GlimpseConfiguration.GetConfiguredMessageBroker() == null)
+            {
+                return true;
+            }
+            if (GlimpseConfiguration.GetRuntimePolicyStategy()() == RuntimePolicy.Off) 
+            {
+                return true;
+            }
+            return false;
+        }
+
```

Comment:
This is a fallback from my old Pascal programming days and wanting to define functions before they are called. lol.